server/bin/unittests

```diff
@@ -462,6 +476,7 @@ function _reset_state {
 
     export PATH=${_orig_PATH}
     unset CONFIG
+    unset LOGSDIR
```

Comment:
We also need to remove the state config file we moved to standard location below.

Reply:
We blow away the whole directory, so there is no need to deal with individual files, is there?

Reply:
We want to make sure we clean up after ourselves in the unit test harness so that we can tell when the server leaves unexpected files around due to its operation.